packages/collector/test/tracing/database/redis/connect-via/cluster.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * (c) Copyright IBM Corp. 2024
+ */
+
+'use strict';
+
+const MAX_TRIES = 50;
+const { delay } = require('../../../../../../core/test/test_util');
+
+// node bin/start-test-containers.js --redis-node-0 --redis-node-1 --redis-node-2
+// docker exec -it 2aaaac7b9112 redis-cli -p 6379 cluster info
+module.exports = async function connect(redis, log, tries = 0) {
+  // NOTE: we cannot run redis cluster on Tekton https://github.com/bitnami/charts/issues/28894
```

Comment:
The comments are little confusing, how can we run the tests in local?

Reply:
> We use the cluster from Azure for localhost and Tekton.

https://github.com/instana/nodejs/pull/1270/files#diff-0ded018ffc0132683fbe045fceb5018de0890e0b2a318bb2bcb801ddaffca85cR16

I will update the comment section to make it more clear 👍 
The docker command is the initial command I have used.

Reply:
>  // Please set the environment variables to connect to the cloud redis cluster:
>  //    export AZURE_REDIS_CLUSTER=team-nodejs-redis-cluster-tekton.redis.cache.windows.net:6380
>  //    export AZURE_REDIS_CLUSTER_PWD=